tests/end2end-tests/graphbolt-gs-integration/graphbolt-graph-construction.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+trap cleanup SIGINT SIGTERM ERR EXIT
+
+usage() {
+  cat <<EOF
+Usage: $(basename "${BASH_SOURCE[0]}") [-h] [-x] [-i /path/to/ml-100k] [-o /path/to/output/data]
+
+Converts the raw ml-100k data to format used for integration tests
+
+Available options:
+
+-h, --help          Print this help and exit
+-x, --verbose       Print script debug info (set -x)
+-i, --ml100k-path   Path to the processed ml-100k data, created by tests/end2end-tests/data_gen/process_movielens.py
+-o, --output-path   Path under which the converted data will be created.
+
+EOF
+  exit
+}
+
+# Parse command-line arguments
+parse_params() {
+    # Default values for input and output paths
+    INPUT_PATH="/storage/ml-100k"
```

Comment:
The ml-100k is stored in `storage/`?

Reply:
That's the hardcoded assumption in all our tests, the raw dataset is baked into the CI image at that location, which is then processed and output to `/data/ml-100k/`, see https://github.com/awslabs/graphstorm/blob/f7796f29402948d4e477bd8adb491a41db5195a3/tests/end2end-tests/create_data.sh#L6